src/test/resources/overseas_entity_v_3_2.json

```diff
@@ -745,6 +745,7 @@
   },
   "is_remove" : true,
   "is_secure_register" : true,
+  "who_is_registering" : "agent",
```

Comment:
if we remove this field to replicate records that are currently on mongo then the test fails as it is unable to read and convert from mongo, I believe we still need to be able to read these exesiting records from mongo

Reply:
Test failed due to me removing the field, the failure was thrown by a null checker which does not allow nulls in the test data. My concern was addressed as the older versions of the schema json test fails converted ok, and they do not have this new field.